### DISCUSSION TITLE
Notes from the Laboratory: March 2023

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#216] [Comments]
# Notes from the Laboratory: March 2023

Another monthly report from the depths of the laboratory of kollitsch.dev - March 2023

https://kollitsch.dev/blog/2023/notes-from-the-laboratory-march/

**Comment by brycewray** [comment]
FWIW, my aging eyes find a light theme a lot easier to read — other than how giscus is currently handling it, of course. :-) Perhaps when you go back to the dark theme, you could add a little more contrast to the body text?

**Comment by davidsneighbour** [comment]
I am having issues giving the theme the proper colors currently. It's a mix of the bootstrap default, and everywhere I defined SASS variables, it's properly done. It appears though that Bootstrap either does NOT copy the SASS variables over to the CSS variables or I messed it up somewhere in between. Which is why I deleted a lot of SCSS. Getting the color scheme right is prio 1 ;) You see the blue at some places, that is where the Bootstrap defaults bleed through. The contrast will be AAA in the end (at least a 7.0 contrast ratio).

**Comment by davidsneighbour** [comment]
> Perhaps when you go back to the dark theme, you could add a little more contrast to the body text?

I think that is something they fixed with Bootstrap 5.3-alpha3. The calc()ulation of contrast colors was throwing some warnings about not being able to find a proper color to contrast the background colors.
